src/media/local-video-track.ts

```diff
@@ -0,0 +1,75 @@
+import { LocalTrack } from './local-track';
+
+export enum FacingMode {
+  user = 'user',
+  environment = 'environment',
+}
+
+export enum OptimizationMode {
+  motion = 'motion',
+  detail = 'detail',
+}
+
+export enum DisplaySurface {
+  browser = 'browser',
+  monitor = 'monitor',
+  window = 'window',
+}
+
+type BaseVideoConstraints = Pick<
+  MediaTrackConstraints,
+  'aspectRatio' | 'height' | 'width' | 'frameRate' | 'deviceId'
+>;
+
+type BaseDisplayConstraints = Pick<
+  MediaTrackConstraints,
+  'aspectRatio' | 'height' | 'width' | 'frameRate' | 'suppressLocalAudioPlayback'
+>;
+
+export interface CameraConstraints extends BaseVideoConstraints {
```

Comment:
Is `BaseVideoConstraints` used anywhere else? Looks like it's not needed (same for `BaseDisplayConstraints`) and `CameraConstraints` and `DisplayConstraints` can just be defined from the constraints directly?

Reply:
pick does not work with extends , so i have to define it before adding other values to the interface

@marcin-bazyl  had the same question the only way is to do 

`export interface CameraConstraints extends xxx` now

Reply:
I meant that I'm not sure the repetition is that bad, thinking it might be better to just do:
```
type CameraConstraints = Pick<
  MediaTrackConstraints,
  'aspectRatio' | 'height' | 'width' | 'frameRate' | 'deviceId' | 'facingMode'
>;

type DisplayConstraints = Pick<
  MediaTrackConstraints,
  'aspectRatio' | 'height' | 'width' | 'frameRate' | 'deviceId' | 'displaySurface' | 'logicalSurface'
>;
```
(note: I don't see any reference to 'optimizationMode' in the spec?)